mappings/net/minecraft/client/render/block/entity/ChestBlockEntityRenderer.mapping

```diff
@@ -1,2 +1,7 @@
 CLASS net/minecraft/class_826 net/minecraft/client/render/block/entity/ChestBlockEntityRenderer
 	FIELD field_4365 isChristmas Z
+	METHOD method_22749 (Lnet/minecraft/class_4587;Lnet/minecraft/class_4588;Lnet/minecraft/class_630;Lnet/minecraft/class_630;Lnet/minecraft/class_630;FII)V
+		ARG 1 matrices
+		ARG 2 vertices
```

Comment:
`vertexConsumer`

Reply:
If it's a VertexConsumer naming it `vertices` is intended.